_posts/2018-08-08-cockpit-175.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 175
+author: mvo
+date: 2018-08-08
+tags: cockpit linux
+slug: cockpit-175
+category: release
+summary: Network bound disk encryption
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 175.
+
+### Network bound disk encryption
```

Comment:
The section heading "Network bound disk encryption" does not accurately describe the contents of the section.  NBDE is only one subfeature of the ability to manage LUKS

Reply:
Excellent point!

Perhaps "Storage: Key management for encrypted devices" might be a better fit?

Reply:
Very good!